src/utils.ts

```diff
@@ -334,7 +335,13 @@ export function serializeReservations(reservations: Reservation[]) {
   if (current.length > 0) {
     line += `${currentHeader}\n`;
     line += `${reservationTitles}\n`;
-    current.map(reservation => line += serializeReservation(reservation));
+    current.map(reservation => {
+      if (serializeReservationHandler) {
+        line += serializeReservationHandler(reservation);
+      } else {
+        line += serializeReservation(reservation);
+      }
+    });
   }
 
   if (current.length > 0 && upcoming.length > 0) {
```

Comment:
Can we nest this `current.length` conditional into the below `upcoming.length`? It will cut down on the repeat check for `upcoming.length` and fits the logical flow a bit better to me.